grafana/src/eaRelease.jsonnet

```diff
@@ -0,0 +1,276 @@
+// https://grafana.com/docs/grafana/latest/dashboards/json-model/
+local grafana = import 'grafonnet/grafana.libsonnet';
+local graphPanel = grafana.graphPanel;
+local timeseries = grafana.timeseries;
+local textPanel = grafana.text;
+local prometheus = grafana.prometheus;
+
+local shared = import './shared.libsonnet';
+local addSideLegend = shared.helpers.addSideLegend;
+
+local cortexDataSource = shared.constants.cortexDataSource;
+local instanceFilter = shared.constants.instanceFilter;
+local interval = shared.constants.interval;
+
+local templates = shared.createTemplates(multiService=true);
+
+/**
+ * Panels
+ */
+local inboundRequestsNodeEAPanel = textPanel.new(
+    content='# Inbound Requests (Node → EA)',
+);
+
+local ErrorRate5xxPanel = graphPanel.new(
+  title='Top 10 - 5xx Error Rate',
+  datasource=cortexDataSource,
+  format='percent',
+).addTarget(
+  prometheus.target(
+    'topk(10, sum by(app_name) (rate(http_requests_total{' + instanceFilter + ',status_code=~"^5..",app_name=~"$adapter.*"}' + interval + ')) / sum by(app_name) (rate(http_requests_total{app_name!="",' + instanceFilter + ',app_name=~"$adapter.*"}' + interval + ')) * 100)'
```

Comment:
Check out this query in the dashboard, because it uses the nginx metric instead of the adapter one; the former is much more reliable

Reply:
I think we changed it to be http_requests_total instead of nginx because this version is supposed to be for the node operators to make, and they don't have nginx set up